pkgs/tools/misc/ocs-url/default.nix

```
@@ -0,0 +1,53 @@
+{ stdenv, fetchgit, libsForQt5, git, lib }:
+
+stdenv.mkDerivation rec {
+  name = "ocs-url";
+  version = "3.1.0";
+
+  meta.description = "Open Collaboration System for use with DE store websites";
+  meta.license = lib.licenses.gpl3Only;
+  meta.maintainers = [ lib.maintainers.SohamG ];
+  meta.platforms = lib.platforms.linux;
+  srcs = [
+    (fetchgit {
+      url = "https://www.opencode.net/dfn2/ocs-url.git";
+      rev = "refs/tags/release-3.1.0";
```

[untested] It's customary to use tags without full qualifier. While at it you can reuse version from package version:

```suggestion
      rev = "release-${version}";
```